src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -95,5 +90,19 @@ public abstract class AnalyzeOptionsBase : CommonOptionsBase
                        "should be emitted to the console and log file (if appropriate). " +
                        "Valid values: ScanTime.")]
         public virtual IEnumerable<string> Traces { get; set; }
+
+        [Option(
+            "level",
+            Separator = ';',
+            Default = new FailureLevel[] { FailureLevel.Error, FailureLevel.Warning },
+            HelpText = "Filter results. Valid values: None, Note, Warning, and Error.")]
+        public IEnumerable<FailureLevel> Level { get; set; }
+
+        [Option(
+            "kind",
+            Separator = ';',
+            Default = new ResultKind[] { ResultKind.Fail },
+            HelpText = "Filter results. Valid values: None, NotApplicable, Pass, Facil, Review, Open, and Informational")]
```

Comment:
>HelpText = "Filter results. Valid values: None, NotApplicable, Pass, Facil, Review, Open, and Informational")] [](start = 12, length = 110)

"Filter output one or more result kinds. Valid values: Fail (for literal scan results), Pass, Review, Open, NotApplicable and Informational."

Reply:
Done.

---
In reply to: [559751125](https://github.com/microsoft/sarif-sdk/pull/2241#discussion_r559751125) [](ancestors = 559751125)